src/components/tasks/card/index.tsx

```diff
@@ -20,11 +20,12 @@ const Card: FC<Props> = ({ content }) => {
     title,
     endsOn,
     startedOn,
-    ownerId,
     status,
+    assignee,
   } = content;
 
-  const ownerProfilePic = `${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}${ownerId}/img.png`;
+  const [assigneeProfilePic, setAssigneeProfilePic] = useState(`${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}${assignee}/img.png`);
```

Comment:
NIT: just for readability, can we store the value under `useState` in a variable and then use that variable here?

Reply:
Yes, we can,
But as it is already `assigneeProfilePic`, and we will need a similar name for the value, which can be confusing.
Let's handle this later 😃 

Thanks for the review 😄